code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react"
```

Comment:
It would be nice to have a short description of your project in the README-file.

Reply:
Done, thanks :)